### ISSUE TITLE
msco server does not check for None in certain circumstances

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#498]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/498) by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

Server log shows, e.g.  

```
Traceback (most recent call last):
  File "/home/mss-demo/miniconda3/envs/stable/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/mss-demo/miniconda3/envs/stable/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/mss-demo/miniconda3/envs/stable/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/mss-demo/miniconda3/envs/stable/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/mss-demo/miniconda3/envs/stable/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/mss-demo/miniconda3/envs/stable/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/mss-demo/miniconda3/envs/stable/lib/python3.7/site-packages/mslib/mscolab/server.py", line 88, in wrapper
    return func(*args, **kwargs)
  File "/home/mss-demo/miniconda3/envs/stable/lib/python3.7/site-packages/mslib/mscolab/server.py", line 214, in add_permission
    u_id = User.query.filter_by(username=username).first().id
AttributeError: 'NoneType' object has no attribute 'id'
WARNING:engineio.server:Invalid session 2edca1db05de425da9bbf418bc3eb233
```

‌

**Comment by ReimarBauer** [comment]
**Original comment by Shivashis Padhi (Bitbucket: [plant99](https://bitbucket.org/plant99), GitHub: [plant99](https://github.com/plant99)).**

----------------------------------------

Currently, the auth token expires in 10 minutes. But this PR fixes that  
[https://bitbucket.org/wxmetvis/mss/pull-requests/744/fixes-issue-491-issue-483/diff#Lmslib/mscolab/models.pyT61](https://bitbucket.org/wxmetvis/mss/pull-requests/744/fixes-issue-491-issue-483/diff#Lmslib/mscolab/models.pyT61)  
  
It’d be 10 days from next release.

**Comment by ReimarBauer** [comment]
**Original comment by Shivashis Padhi (Bitbucket: [plant99](https://bitbucket.org/plant99), GitHub: [plant99](https://github.com/plant99)).**

----------------------------------------

@{557058:fabca9ad-f8cf-4f48-b9aa-8ab5c2f5d433} should I mark this issue close?

**Comment by ReimarBauer** [comment]
**Original comment by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

Resolved by pull request #744
